src/main/java/io/reactivex/internal/operators/maybe/MaybeSwitchIfEmptyToMaybe.java

```diff
@@ -24,11 +24,11 @@
  *
  * @param <T> the value type
  */
-public final class MaybeSwitchIfEmpty<T> extends AbstractMaybeWithUpstream<T, T> {
+public final class MaybeSwitchIfEmptyToMaybe<T> extends AbstractMaybeWithUpstream<T, T> {
```

Comment:
I'd keep the original name, less change per PR, and name the new version `MaybeSwitchIfEmptySingle`.